checker/bin-devel/javac

```diff
@@ -24,13 +24,12 @@ binaryDir="${mydir}"/../dist
 javacJar=${binaryDir}/javac.jar
 
 cfDir="${mydir}"/../..
-annoToolsDir="${cfDir}"/../annotation-tools
```

Comment:
I think this change means that running the `compileJava` in AFU and CF doesn't work; it is necessary to do `./gradlew assemble` if AFU might have changed.  Is that correct?

Reply:
Oh, I can put that back.  It only included the class files from scene-lib before and used the wrong location:
```
"${annoToolsDir}"/scene-lib/bin
```

I can fix it so that it is the correct location.

Reply:
I fixed this.